src/index.js

```diff
@@ -0,0 +1,96 @@
+const state = {
+    tempVar : 76,
+    skyDisplay : {
+        'Sunny' : '☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️',
+        'Cloudy' : '☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️',
+        'Rainy' : '🌧🌧🌧🌧🌧🌧🌧🌧🌧🌧',
+        'Stormy' : '⛈⛈⛈⛈⛈⛈⛈⛈⛈⛈',
+        'Snowy' : '🌨🌨🌨🌨🌨🌨🌨🌨🌨🌨',
+        'Gay' : '🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈',
+        'Meatball' : '🍝🍝🍝🍝🍝🍝🍝🍝🍝🍝'
+    },
+    landscapeDisplay : {
+        'Hot' : '🌵🌵🌵🌵🌵🌵🌵🌵🌵🌵',
+        'Warm' : '🌴🌴🌴🌴🌴🌴🌴🌴🌴🌴',
+        'Meh' : '🍂🍂🍂🍂🍂🍂🍂🍂🍂🍂',
+        'Cold' : '❄️❄️❄️❄️❄️❄️❄️❄️❄️❄️',
+        'Freezing' : '☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️'
+    },
+    cityName : ''
+};
+
+const upTemp = (event) => {
+    state.tempVar += 1;
+    const tempDisplay = document.getElementById("temp-display-big");
+    tempDisplay.textContent = `${state.tempVar} F`;
+    tempChecker(state.tempVar, tempDisplay);
+};
+
+const downTemp = (event) => {
+    state.tempVar -= 1;
+    const tempDisplay = document.getElementById("temp-display-big");
+    tempDisplay.textContent = `${state.tempVar} F`;
+    tempChecker(state.tempVar, tempDisplay);
+};
+
+const skyDisplayer = (event) => {
+    const skyDisplayElement = document.getElementById('sky-display');
+    const skyType = document.getElementById('sky-type');
+    const selection = skyType.value
+    skyDisplayElement.textContent = `${state.skyDisplay[selection]}`;
+    if (skyType.value === 'Gay') {
+        document.body.style.background = 'linear-gradient(45deg, red, orange, yellow, green, blue, indigo, violet, red)';
+    } else {
+        document.body.style.background = 'gray';
+    }
+};
+
+const tempChecker = (temp, tempDisplay) => {
+    const landscapeDisplay = document.getElementById('landscape-display');
+    if (temp > 80) {
+        tempDisplay.style.color = 'red';
+        landscapeDisplay.textContent = state.landscapeDisplay['Hot'];
+    } else if ((temp <= 80) && (temp > 60)) {
```

Comment:
Notice that we don't actually need to check the upper bound of the range. If we reached this check, then the prior check for whether the temp was above 80 must have failed, meaning it's no more than 80.